src/documentation/pages/components/footer/examples/index.vue

```diff
@@ -0,0 +1,46 @@
+<template lang="pug">
+  div
+    h2 Examples
+    p.-text
+      | To use footers, apply the class <code>chi-footer</code> to <code>&lt;footer&gt;</code>
+      | and wrap all content in <code>chi-footer__content</code>.
+    <div v-if="['lumen', 'portal', 'brightspeed'].includes($store.state.themes.theme)">
```

Comment:
You are mixing HTML with Pug, we can leave everything as Pug, please apply this to others.
```suggestion
    div(v-if="['lumen', 'portal', 'brightspeed'].includes($store.state.themes.theme)")
```